request.go

```diff
@@ -31,17 +31,43 @@ type Response struct {
 // uriForAPI is to be called with something like "/v1/events" and it will give
 // the proper request URI to be posted to.
```

Comment:
Could you please amend the docstring of this function to cover the change done below?